### DISCUSSION TITLE
Why food log needs access to profile?

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#4] [Q&A] [answered]
Jlai Great work on this project. Thx.

However I am concerned with amount of access app requires. I am only interested in tracking my food, why do I need to give access to my profile?

Thanks.

**Comment by jlai** [comment]
It's a reasonable concern. There's a number of [user settings](https://dev.fitbit.com/build/reference/web-api/user/get-profile/) in the profile.

I'm planning on adding local settings for unit settings so it doesn't need profile access (which also should make the site faster, because it doesn't have to check).

**Comment by jlai** [comment, marked as answer]
I've removed the profile permission requirement. If you sign in without profile permission, you'll need to manually set the units in the settings page or else you'll get metric. (TODO: add a notice to the user to go set it up)
